src/metadata/routes/avus.clj

```diff
@@ -79,4 +91,13 @@ then all remaining AVUs in the request will be added to the target item."
            :description "
 Copies all Metadata Template AVUs from the data item with the ID given in the URL to other data
 items sent in the request body."
-           (ok (avus/copy-avus user target-type target-id body)))))
+           (ok (avus/copy-avus user target-type target-id body)))
+
+    (DELETE "/:target-type/:target-id/:avu-id" []
```

Comment:
Actually, maybe this endpoint doesn't make sense, since all AVU IDs are UUIDs, so there's no need to also require the target type and ID.

It should probably just be `DELETE /avus/:avu-id`, but since there are currently no plans to use this endpoint (`POST /avus/deleter` is the one required for communties), I think I'll just remove this cruft now until we need it.